enterprise_gateway/services/processproxies/yarn.py

```diff
@@ -159,11 +157,20 @@ def confirm_yarn_queue_availability(self, **kwargs):
 
         self.start_time = RemoteProcessProxy.get_current_time()
         self.candidate_queue = self.resource_mgr.cluster_scheduler_queue(candidate_queue_name)
+
+        if self.candidate_queue is None:
+            self.log.debug("Queue: {} does not exist in the cluster".format(candidate_queue_name))
+            return
+
         self.candidate_partition = self.resource_mgr.cluster_queue_partition(self.candidate_queue, node_label)
 
-        self.log.debug("Checking endpoint: {} if partition: {} has used capacity <= {}%".format(self.yarn_endpoint,
-                                                                                                self.candidate_partition,
-                                                                                                    partition_availability_threshold))
+        if self.candidate_partition is None:
+            self.log.debug("Parition: {} not found in {} queue".format(node_label, candidate_queue_name))
```

Comment:
See previous comment - log as warning, add suffix statement.

Reply:
Need to fix typo in 'Partition' as well.

Reply:
Commited the changes.